<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_ChangePassword</class>
 <widget class="QDialog" name="Dlg_ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:rgb(255,255,255);
}
QPushButton
{
background:rgba(0,0,0,0);

}
QPushButton#pushButton_2
{
image:url(:/image/home.png);
}
QPushButton#pushButton_2:hover
{
image:url(:/image/home_pressed.png);

}
QPushButton#enter_button
{
image:url(:/image/login.png);
color:white;
}
QPushButton#enter_button:hover
{
image:url(:/image/login-pressed.png);
color:white;
}
QRadioButton
{

}
QPushButton#pushButton
{
color:white;
}
QPushButton#pushButton:hover
{
color:white;
}
QLineEdit
{
border:none;
border-bottom:1px solid rgb(229,229,229);
font:normal 12px;
color:rgb(154,154,154);
}
QLineEdit:hover
{
border:none;
border-bottom:1px solid rgb(0,0,0);
font:normal 12px;
color:rgb(154,154,154);
}
QLineEdit:focus
{
border:none;
border-bottom:1px solid rgb(0,0,0);
font:normal 12px;
color:rgb(0,0,0);
}
</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>430</width>
     <height>220</height>
    </rect>
   </property>
   <widget class="QPushButton" name="enter_button">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>180</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确    认</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input_2">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>50</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input_1">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>10</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_input">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>130</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nickname_input">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>90</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>重置密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="user_button">
       <property name="text">
        <string>用    户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="manager_button">
       <property name="text">
        <string>管理员</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>430</width>
     <height>330</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Logo.qrc">:/image/etherstrike.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>20</y>
     <width>70</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>widget_2</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton_2</tabstop>
  <tabstop>password_input_1</tabstop>
  <tabstop>password_input_2</tabstop>
  <tabstop>nickname_input</tabstop>
  <tabstop>account_input</tabstop>
  <tabstop>user_button</tabstop>
  <tabstop>manager_button</tabstop>
  <tabstop>enter_button</tabstop>
 </tabstops>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
